views/index.ejs

```diff
@@ -54,7 +54,7 @@
                     <li>Body: fat</li>
                 </ul>
                 <p>If you don't have data for some of those fields (eg you don't keep a food log), they will just be blank.</p>
-                <p>For more detail, see the <a href="https://wiki.fitbit.com/display/API/Fitbit+Resource+Access+API">Fitbit API documentation.</a></p>
```

Comment:
Ah, okay. As long as it goes to where it needs to go. I just didn't even try to make an account and check out the page. I'll close this.